ignite/handlers/wandb_logger.py

```diff
@@ -26,7 +26,7 @@ class WandBLogger(BaseLogger):
     Args:
         args: Positional arguments accepted by `wandb.init`.
         kwargs: Keyword arguments accepted by `wandb.init`.
-            Please see `wandb.init <https://docs.wandb.ai/ref/python/init>`_ for documentation of possible parameters.
+            Please see `wandb.init <https://docs.wandb.ai/ref/python/sdk/functions/init/>`_ for documentation of possible parameters.
```

Comment:
Let's put this into another PR

Reply:
https://github.com/pytorch/ignite/pull/3443